src/hooks/video/agora/useAgoraScreenShare.ts

```diff
@@ -24,10 +24,43 @@ export const useAgoraScreenShare: ReactHook<
   const [localScreenTrack, setLocalScreenTrack] = useState<ILocalVideoTrack>();
   const [localAudioTrack, setLocalAudioTrack] = useState<ILocalAudioTrack>();
 
+  const stopShare = useCallback(async () => {
+    if (!venueId || !userId || !client) return;
+
+    updateTalkShowStudioExperience({
+      venueId,
+      userId,
+      experience: {
+        isSharingScreen: false,
+      },
+    });
+
+    localScreenTrack?.stop();
+    localScreenTrack?.close();
+
+    localAudioTrack?.stop();
+    localAudioTrack?.close();
+
+    await client.unpublish();
+
+    setLocalScreenTrack(undefined);
+    setLocalAudioTrack(undefined);
+  }, [venueId, userId, client, localAudioTrack, localScreenTrack]);
+
+  const leaveChannel = useCallback(async () => {
+    stopShare();
+    await client?.leave();
```

Comment:
```suggestion
  const leaveChannel = useCallback(async () => {
    if(!client) return;
    
    stopShare();
    await client.leave();
```

Reply:
@yurii-lubynets @dmitrii-zolotukhin Do we actually want to not call `stopShare()` if there is no `client`?

Or do we only want to avoid calling `client.leave()` when there is no `client`?